github/pulls_reviews.go

```diff
@@ -0,0 +1,14 @@
+package github
+
+import "time"
+
+// No docs are out for this quite yet.
+type PullRequestReview struct {
```

Comment:
Nothing is currently using `Review` name. Have you considered using that, instead of `PullRequestReview`? Since that's what the JSON field is called. Plus it's documented as "The review that was affected", rather "The pull request review that was affected".

Or do you expect that there may be other types of reviews in the future, so it's better to proactively name this `PullRequestReview` to avoid future collisions?